web/src/app/components/job-list-item/job-list-item.component.ts

```diff
@@ -40,19 +131,34 @@ export class JobListItemComponent implements OnInit, OnDestroy {
   jobPinUrl: SafeUrl;
   readonly jobListItemActionsType = JobListItemActionsType;
   subscription: Subscription = new Subscription();
+  loiData = new Map<string, string[]>([]);
```

Comment:
Is the item component only used in JobList? If so, can we move it below side-panel/job-list?